node/node-requesttracker-rest.py

```diff
@@ -45,7 +45,7 @@ def parse_ticket_lines(lines):
     if not history:
         query = query + " AND ( Status = 'new' OR Status = 'open' )"
     query = quote(query)
-    search = Request(uri + "/REST/1.0/search/ticket/" + '?query=' + query)
+    search = Request(uri + "/REST/1.0/search/ticket/" + '?query=' + query, headers={"Referer":uri})
```

Comment:
```suggestion
    search = Request(uri + "/REST/1.0/search/ticket/" + '?query=' + query, headers={"Referer": uri})
```

Is it more readable? 🤷 

But at least it's PEP8 compliant 😋